frontend/src/components/admin/MinespaceUserList.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { Table, Button, Popconfirm, Icon } from "antd";
+import PropTypes from "prop-types";
+import CustomPropTypes from "@/customPropTypes";
+import * as Strings from "@/constants/strings";
+import NullScreen from "@/components/common/NullScreen";
+
+const propTypes = {
+  minespaceUsers: PropTypes.arrayOf(CustomPropTypes.minespaceUser),
+  mines: PropTypes.arrayOf(CustomPropTypes.mineName),
+  handleDelete: PropTypes.func,
+};
+
+const defaultProps = {
+  minespaceUsers: [],
+  mines: [],
+  handleDelete: null,
```

Comment:
null is not a valid default prop for a function, 
use: 

```
handleDelete:  ()=> {};
```

Reply:
thanks, fixed.